<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="5">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QPushButton" name="btnsv">
         <property name="text">
          <string>save data as csv file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="savebtn">
         <property name="text">
          <string>save a graphic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ruslbtn">
         <property name="text">
          <string>save ruslt of control </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>давления,бар</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLCDNumber" name="parN">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="prbar">
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Serial</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comL"/>
       </item>
       <item>
        <widget class="QPushButton" name="openbtn">
         <property name="text">
          <string>OPEN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closebtn">
         <property name="text">
          <string>CLOSE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkB">
         <property name="text">
          <string>checkHere to open</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3" colspan="3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>control import</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="linet"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnw">
         <property name="text">
          <string>time, c</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linep"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnp">
         <property name="text">
          <string>давления,бар</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>Write the path of file</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLineEdit" name="lineinput">
         <property name="text">
          <string>E:\import_data.csv</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inputF">
         <property name="text">
          <string>Enter the data control</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2" colspan="6">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Graphic</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="PlotWidget" name="grf" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
